app/javascript/packages/document-capture/components/review-issues-step.jsx

```diff
@@ -107,12 +107,14 @@ function ReviewIssuesStep({
               errorMessage={selfieError ? <FormErrorMessage error={selfieError} /> : undefined}
             />
           ) : (
-            <SelfieCapture
-              ref={registerField('selfie', { isRequired: true })}
-              value={value.selfie}
-              onChange={(nextSelfie) => onChange({ selfie: nextSelfie })}
-              errorMessage={selfieError ? <FormErrorMessage error={selfieError} /> : undefined}
-            />
+            <div style={{ maxWidth: '375px' }}>
```

Comment:
> Inline styles on a wrapper element is not an ideal implementation here, though it seemed more appealing than one of the obvious alternatives:
> 
> * BEM modifier class `selfie-capture--size-as-id-input`, applied either via prop `className` or `isSizedAsIdInput` prop on `SelfieCapture`.
> * `SelfieCapture` supports `style` prop, applying inline style instead of wrapper element.

Instead of BEM style, could we just define something like `.image-input`? and apply that to all 3 input fields? Then it would have the `max-width` bit?

Reply:
I don't think the style attribute will be recognized by the CSP. This would need to be nonced in order to work.

Reply:
> I don't think the style attribute will be allowed by the CSP. This would need to be nonced in order to work.

Hm, I was a bit confused by this, since we have inline styles elsewhere already, notably in the [`AcuantCaptureCanvas` component](https://github.com/18F/identity-idp/blob/master/app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx), which appear to work just fine. 

I dug into this a bit more, and it appears that it's possibly fine because of how React is actually assigning these styles using the [style property](https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style).

Related: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src

>However, styles properties that are set directly on the element's style property will not be blocked, allowing users to safely manipulate styles via JavaScript:
>
>```js
>document.querySelector('div').style.display = 'none';
>```

That being said, I like @zachmargolis's suggestion of a class here anyways, so I'll plan to move away from the inline styles here. It might be worth considering for `AcuantCaptureCanvas` as well, just to avoid any potential future conflicts with CSP.